src/backend/crud/agent.py

```diff
@@ -0,0 +1,82 @@
+from sqlalchemy.orm import Session
+
+from backend.database_models.agent import Agent
+from backend.schemas.agent import UpdateAgent
+
+
+def create_agent(db: Session, agent: Agent) -> Agent:
+    """
+    Create a new agent.
+
+    Args:
+      db (Session): Database session.
+      agent (Agent): Agent to be created.
```

Comment:
Can you add a bit more context here? Maybe one more sentence describing what an agent is